client/signup/steps/survey/index.jsx

```diff
@@ -147,7 +147,7 @@ export default React.createClass( {
 				category_label: label
 			} );
 		}
-		SignupActions.submitSignupStep( { stepName: this.props.stepName }, [], { surveySiteType: this.props.surveySiteType, surveyQuestion: vertical.value } );
+		SignupActions.submitSignupStep( { stepName: this.props.stepName }, [], { surveySiteType: this.props.surveySiteType, surveyQuestion: this.state.stepOne.value } );
```

Comment:
Juuuuust for safety, and in case someone decides to come in here later and change `isSurveyOneStep`, could you duplicate this line, putting the new version in the `if` block just above and the old version in `else` clause?

The reason being that if `isSurveyOneStep` is true, `this.state.stepOne` will be `null` and this line will throw a fatal error.

Really, we should just remove all the one-step specific code in this file, but until we do I'd prefer not to add the risk of any fatal errors.